ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,73 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
-
+import { openSuccess } from '../alerts/index'
+ 
 const actions = {
   INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_GET_ALL_PENDING: 'inventory/get_all_PENDING',
+  INVENTORY_CREATE: 'inventory/create',
+  INVENTORY_DELETE: 'inventory/delete',
+  INVENTORY_REFRESH: 'inventory/refresh'
 }
 
 export let defaultState = {
-  all: []
+  all: [],
+  fetched: false
 }
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
-})
+export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => 
+  (dispatch, getState, config) => axios
+    .get(`${config.restAPIUrl}/inventory`)
+    .then((suc) => {
+      dispatch(refreshInventory(suc.data))
+      dispatch(openSuccess("INVENTORY_GET_ALL Success"))
```

Comment:
Rename all `openSuccess` alerts to a more user-friendly message (e.g., "INVENTORY_GET_ALL Success" could be changed to "Retrieved all inventories", removing the action name with some description).

Reply:
Changes made in commit 964580f8b9c107fc99b66c123075ac13383a93eb